### ISSUE TITLE
Install fails, Uninstall fails

### ISSUE THREAD

**Issue by zopefiend** [MicrosoftDocs/sql-docs#5889]
How about some troubleshooting gidance when the following the listed steps fails?
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5559d56d-e787-2c02-970d-a2825911d81e
* Version Independent ID: 99f7b922-9fca-dc06-f934-d4a87c098764
* Content: [Uninstall existing instance - SQL Server](https://docs.microsoft.com/en-us/sql/sql-server/install/uninstall-an-existing-instance-of-sql-server-setup?view=sql-server-ver15&tabs=Windows10)
* Content Source: [docs/sql-server/install/uninstall-an-existing-instance-of-sql-server-setup.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/sql-server/install/uninstall-an-existing-instance-of-sql-server-setup.md)
* Product: **sql**
* Technology: **install**
* GitHub Login: @cawrites
* Microsoft Alias: **chadam**

**Comment by WilliamAntonRohm** [comment]
@zopefiend -- thank you again for your feedback.  Please see the resources in the response to #5888.  

@cawrites -- please look into this potential documentation issue.

**Comment by MashaMSFT** [comment]
Hi @zopefiend , thanks for your feedback! There is a section for in the [event of a failure](https://docs.microsoft.com/en-us/sql/sql-server/install/uninstall-an-existing-instance-of-sql-server-setup?view=sql-server-ver15&tabs=Windows10#in-the-event-of-failure): 

![image](https://user-images.githubusercontent.com/32783170/104384046-ad7fa600-54e5-11eb-9fc4-116a37071ddd.png)

Please let us know how we can modify this information to better assist you.

Thanks!
Masha from the SQL Docs team

**Comment by cawrites** [comment]
@zopefiend we have provided a link that provides guidance in troubleshooting failed installs. Although it is impossible to cover every scenario the KB lists some primary methods for troubleshooting. Your initial ask was for guidance, so we have provided that link for review. If you are having a specific error related to an install/uninstall here is a list of resources that may be helpful to you.
https://docs.microsoft.com/en-us/sql/sql-server/sql-server-get-help?view=sql-server-ver15
Thank you for contacting  Microsoft.
